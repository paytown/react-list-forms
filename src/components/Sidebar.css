.Sidebar {
  padding: 40px;
  display: flex;
  flex-direction: column;
  height: calc(100vh - 80px);
  background-color: purple;

  a {
    font-size: 8em;
    line-height: 1.2em;
  }
}
