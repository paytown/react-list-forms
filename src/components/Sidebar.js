import React from 'react';
import PropTypes from 'prop-types';
import styles from './Sidebar.css';

function Sidebar({ children }){
  return (
    <nav className={styles.Sidebar}>
      {children}
    </nav>
  );
}

Sidebar.propTypes = {
  children: PropTypes.node
};

export default Sidebar;
