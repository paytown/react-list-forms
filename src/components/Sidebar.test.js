import React from 'react';
import { shallow } from 'enzyme';
import Sidebar from './Sidebar';

describe('sidebar', () => {
  it('makes a sidebar', () => {
    const wrapper = shallow(<Sidebar />);
    expect(wrapper).toMatchSnapshot();
  });
});
