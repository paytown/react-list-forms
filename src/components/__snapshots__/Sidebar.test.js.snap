// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sidebar makes a sidebar 1`] = `
<nav
  className="Sidebar"
/>
`;
